Makefile

```diff
@@ -348,6 +352,10 @@ amdgpu_plugin: criu
 	$(Q) $(MAKE) -C plugins/amdgpu all
 .PHONY: amdgpu_plugin
 
+cuda_plugin: criu
```

Comment:
In contrast to amdgpu plugin, the cuda plugin does not have additional build dependencies.
Would it be possible to add `cuda_plugin` in the list of default targets?
```Makefile
# Default target
all: flog criu lib crit cuda_plugin
```